service-front/assets/js/opg/session-timeout-dialog.js

```diff
@@ -116,24 +116,40 @@ var SessionTimeoutDialog = function (options) {
         this.releaseNavigation();
     };
 
-    this.trapNavigation = function () {
-        continueButton.keydown(function (e) {
-            if (e.key === 'Tab' && e.shiftKey) {
-                e.preventDefault();
-                logoutButton.focus();
+    this.trapNavigation = function (focusables) {
+        var currentElementIndex = 0;
+        focusables[currentElementIndex].focus();
+
+        var lastFocusableIndex = focusables.length - 1;
+
+        this.element.keydown(function (e) {
```

Comment:
It may be out of scope, but a common practice is to close the dialog by using the esc key, can see this here as an example if it fits here https://github.com/ministryofjustice/opg-use-an-lpa/blob/6a0b37c7f4964740b381266f611df03f80b1e951/service-front/web/src/javascript/sessionDialog.js#L92

Reply:
It's not out of scope, I'm just not sure how that key would be interpreted here: we have "OK" and "Sign out" as options, while "Esc" doesn't really signify commitment to either of those, so I'm not sure what it would do. That's what I didn't add a handler for it.

Reply:
Makes sense, in that case maybe its one to discuss separately where we can have a think about the user journey?

Reply:
I suggest raising a separate issue for that as it's not within the scope of this one. I'll do that.

Reply:
@johnnolan I raised https://opgtransform.atlassian.net/browse/LPAL-307 for this if you'd like to expand/refine it.